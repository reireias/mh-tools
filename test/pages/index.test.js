import { mount, createLocalVue, RouterLinkStub } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import target from '@/pages/index'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuetify)

describe('power-or-critical', () => {
  it('should be vue instance', () => {
    const vuetify = new Vuetify()
    const wrapper = mount(target, {
      localVue,
      vuetify,
      stubs: {
        NuxtLink: RouterLinkStub,
      },
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
