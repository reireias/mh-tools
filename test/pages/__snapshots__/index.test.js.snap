// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`power-or-critical should be vue instance 1`] = `"<div class=\\"container\\"><div class=\\"row justify-center\\"><h1 class=\\"primary--text\\">MH Tools</h1></div> <div class=\\"row justify-center\\"><a>攻撃 or 見切り</a></div></div>"`;
